yt/data_objects/selection_objects/data_selection_objects.py

```diff
@@ -184,7 +184,7 @@ def get_data(self, fields=None):
         fluids, particles = [], []
         finfos = {}
         for ftype, fname in fields_to_get:
-            finfo = self.ds._get_field_info(ftype, fname)
+            finfo = self.ds._get_field_info((ftype, fname))
```

Comment:
could consider changing this loop so that it does not split the field tuple in the first place (i.e., `for field in fields_to_get`).

Reply:
yup, definitely. Looks like something I rewrote with a global find & replace, but you're right that the split doesn't make much sense at all now.

Reply:
done